tests/unit/service_test.py

```diff
@@ -439,6 +441,7 @@ def mock_get_image(images):
         raise NoSuchImageError()
 
 
+@pytest.mark.xfail(IS_PLATFORM_WINDOWS, reason='paths use slash')
```

Comment:
we should be able to remove this one once my other PR goes in, well, I hope so, cos that's what I've been trying to fix! :8ball: